tools/roxvet/analyzers/unmarshalreplace/analyzer.go

```diff
@@ -0,0 +1,55 @@
+package unmarshalreplace
+
+import (
+	"go/ast"
+	"go/types"
+	"strings"
+
+	"github.com/stackrox/rox/pkg/set"
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+	"golang.org/x/tools/go/types/typeutil"
+)
+
+const doc = `Direct calls to jsonpb.Unmarshal should be replaced with jsonutil.JSONReaderToProto or jsonutil.JSONBytesToProto`
+
+var Analyzer = &analysis.Analyzer{
+	Name:     "unmarshalreplace",
+	Doc:      doc,
+	Requires: []*analysis.Analyzer{inspect.Analyzer},
+	Run:      run,
+}
+
+var allowedCallerPackages = []string{
+	"github.com/stackrox/rox/migrator/migrations",
```

Comment:
As far as I can see in the code, `jsonpb.Unmarshal` is not used in migration code.
It might be just as good not open for this exception.

Reply:
Good point. I've followed the original comment in the ticket, but did not verify it is actually needed. Removed!